frontend/src/components/sharedStyles/Table/TableHeader.js

```diff
@@ -1,47 +1,46 @@
 import React, { useState } from 'react';
-import {
-    Typography, 
-  } from '@material-ui/core';
+import { Typography, Button, Grid } from '@material-ui/core';
 import { ChevronRightOutlined, ChevronDownOutlined } from 'components/Icons';
+import { CreateButton, SaveButton, EntryComponent } from 'components/sharedStyles/ManageData/CreateSaveButton';
 
 
 function CreateTableHeader(props) {
   const [hiddenTable, setHiddenTable] = useState(true);
+  const [haveCreateButton, setHaveCreateButton] = useState(false);
   function ChangeHiddenTable() {
     setHiddenTable(!hiddenTable) ;
   }
-  /*
-  <Typography
-          variant="h4"
-          component="h1"
-          className={props.headerClassStyle}
-          onClick={ChangeHiddenTable}
-          >{props.title}
-          </Typography>
-          {table}
-          */
-
-    /*
-
-<div>
-        <div onClick={ChangeHiddenTable}
-          >{props.title} {"\n"}
-        </div>
-        {table}
-    */
+  function ChangeButton() {
+    setHaveCreateButton(!haveCreateButton);
```

Comment:
So I like the idea of using a boolean to determine whether we show a specific table/button but I feel like this would be easier to read if we just set the argument to true or false instead of using a bang operator on the hook. 

So it could read `setHaveCreateButton(true); This way we can see in the hook that the state is originally set to false and when the function ChangeButton() is executed it changes the state to true. 

I realize it's a nit pick but this feels like it would be more readable.

Reply:
I would also suggest this for the ChangeHiddenTable() function.